pkgs/top-level/python-packages.nix

```diff
@@ -1350,11 +1350,6 @@ in {
     enablePython = true;
   });
 
-  boost169 = toPythonModule (pkgs.boost169.override {
```

Comment:
Does that mean we can also drop it from all-packages.nix?

Reply:
I don't think so, because python3Packages.boost169 is the python-enabled version of boost169. The normal boost169 is still used a few times. There are also much older boost versions still in use in all-packages.nix.

```console
$ git grep -c boost169
pkgs/applications/emulators/emulationstation/default.nix:2
pkgs/development/compilers/mozart/default.nix:2
pkgs/development/libraries/boost/default.nix:1
pkgs/top-level/all-packages.nix:5
pkgs/top-level/python-packages.nix:1
```

```console
$ git grep -c boost16
pkgs/applications/emulators/emulationstation/default.nix:2
pkgs/applications/networking/instant-messengers/SkypeExport/default.nix:2
pkgs/applications/science/machine-learning/starspace/default.nix:3
pkgs/development/compilers/mozart/default.nix:2
pkgs/development/compilers/osl/default.nix:2
pkgs/development/libraries/boost/default.nix:6
pkgs/development/libraries/clfft/default.nix:2
pkgs/games/freeorion/default.nix:2
pkgs/servers/foundationdb/default.nix:2
pkgs/tools/graphics/appleseed/default.nix:2
pkgs/tools/graphics/yafaray-core/default.nix:2
pkgs/tools/misc/ethminer/default.nix:2
pkgs/top-level/all-packages.nix:32
pkgs/top-level/python-packages.nix:1
```